docs/content/concepts.mdx

```diff
@@ -137,11 +137,11 @@ Learn more about the <UpdateProductNameInLinks link="./configuration-language" n
 
 ## What Is A Store?
 
-A **store** is a <ProductName format={ProductNameFormat.LongForm}/> organization entity that owns an authorization model.
+A "store" is a <ProductName format={ProductNameFormat.LongForm}/> entity used for organizing data.  
 
 </summary>
 
-A store contains all your data, including the [authorization model](#what-is-an-authorization-model) and [relationship tuples](#what-is-a-relationship-tuple).
+Each store contains an [authorization model](#what-is-an-authorization-model) and [relationship tuples](#what-is-a-relationship-tuple).
```

Comment:
```suggestion
Each store may contain an [authorization model](#what-is-an-authorization-model) and [relationship tuples](#what-is-a-relationship-tuple).
```

Reply:
Is there any case where store does not contain an authorization model?

Reply:
Whoops. Sorry I missed your reply.

Yes, when you first create a store it will not contain an authorization model or tuples. Anyway, I don't think this is such a big deal, so we can resolve it.